chart/templates/statsd/statsd-deployment.yaml

```
@@ -57,9 +57,17 @@ spec:
 {{- with .Values.labels }}
 {{ toYaml . | indent 8 }}
 {{- end }}
-{{- if .Values.statsd.extraMappings }}
+{{- if or .Values.statsd.extraMappings .Values.airflowPodAnnotations .Values.statsd.podAnnotations }}
```

```suggestion
{{- if or .Values.statsd.extraMappings .Values.statsd.podAnnotations }}
```

Let's not include the `airflowPodAnnotations`. This isn't an Airflow pod.